testing/constraints-3.11.txt

```diff
@@ -1,24 +1,13 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright 2020 Google LLC
+# Copyright 2024 Google LLC
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     https://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
nit: both `http` and `https` work

Reply:
`http` matches the license [here](https://www.apache.org/licenses/LICENSE-2.0#apply) which could be why we use that in templates such as https://github.com/googleapis/gapic-generator-python/blob/main/gapic/templates/_license.j2